cypress/support/checkin.js

```diff
@@ -0,0 +1,20 @@
+import {
+  Button,
+  Checkbox,
+  MultiColumnList,
+  TextField,
+} from '../../interactors';
+
+Cypress.Commands.add('checkInItem', barcode => {
+  cy.do([
+    TextField({ id: 'input-item-barcode' }).fillIn(barcode),
+    Button('Enter').click(),
+    Button({ text: 'Check in', button: true }).click(),
```

Comment:
Please clarify why for Button the prop `button` should be passed?

Reply:
Is there a "Check in" link as well?

Reply:
yep, menu item in navigation is `Check in`, and button on the screen `Check in`

Reply:
See, this is the reason I get nervous about using anything other than element-ids as selectors. I mean, I'm glad
```
Button({ button: true })
```
works, but without comments explaining why it's there (@NikitaSedyx's comment 1) and what purpose it's serving (comment 2) this line is very confusing to somebody just reading the test. 

Maybe it's just the nature of UI testing that there is tension between identifiers that are unique in the DOM but hidden from users (and therefore dangerous because the test can interact with the DOM in a way the user can't) and selectors that must specify DOM attributes that are hidden from users (and therefore brittle because they may rely on DOM attributes the user never sees and that the developer did not likely intentionally expose as an API surface and therefore are likely to change with notice). 🤷 

Anyway, I think these comments should be copied into the code 🙂

Reply:
You could also scope the interactor which would make more sense. Like say you new that the button was in a certain `Pane`, you could say:

`Pane("Main").find(Button("Check in"))`

which would disambiguate it from the check in button in the menu in a natural feeling way.

Reply:
nice, in our case it's modal, but idea is the same, thanks!